server/services/damsService/queries.js

```diff
@@ -162,9 +163,63 @@ function generateGetAssetsByFileNameQuery(ensembleIndex) {
   };
 }
 
+function generateGetArchiveAssetsQuery() {
```

Comment:
Function to generate the query that retrieves all archive assets from NetX. Currently, there's only about 94 of them.